sdk/helpers/eth.js

```diff
@@ -165,7 +176,7 @@ const eth = {
       value: sendAmount,
       gasPrice: web3.utils.toWei('25', 'gwei'),
       gas: 21000,
-      nonce: await web3.eth.getTransactionCount(from,'pending'),
+      nonce: await eth.getNonce(from),
```

Comment:
we should probably use this function in the other case (line 131), to avoid mismatching values

Reply:
+